meilisearch-types/src/error.rs

```diff
@@ -343,6 +343,7 @@ impl ErrorCode for file_store::Error {
         match self {
             Self::IoError(e) => e.error_code(),
             Self::PersistError(e) => e.error_code(),
+            Self::CouldNotParseFileNameAsUtf8 | Self::UuidError(_) => Code::Internal,
```

Comment:
Does this mean that if the user adds a file with a different name in the update directory they will trigger an internal? Is that an issue?

Reply:
> Does this mean that if the user adds a file with a different name in the update directory they will trigger an internal? 

Yes, but tbh we’ve never seen anyone adding things to the `data.ms`, I’m quite confident on the fact that, if this error happens it’ll be because we effectively wrote something illegal in the first place 😬

Reply:
OK, let's just keep this fact in mind for now and resolve this